### ISSUE TITLE
FS#722 - port forwarding broken

### ISSUE THREAD

**Issue by aparcar** [aparcar/openwrt#762]
*Steve-Newcomb:*

Supply the following if possible:
 - Device problem occurs on

TP-Link Archer c7 ac1750 v.2

 - Software versions of LEDE release, packages, etc.

Reboot (17.01.0, r3205-59508e3)

 - Steps to reproduce
   
Install LEDE.  Overwrite /overlay with something very similar to what
can be found in the attached tar.xz file.  Reboot.  Connect WAN as
PPPoE.  Once it's working, try to forward packets to ports.

A very similar configuration works on OpenWRT 15.01 (Chaos Calmer) on
other v2 units.  Alas, the 5HGz radios don't work on the newer v2
units, so I changed over to LEDE, which seems to work perfectly except
that port forwarding doesn't work on LEDE.  Comparison was made using
the same PPPoE account at the same ISP, and in the same operating
context, with the same forwarding arrangements with all the same IPs.
In other words, it is very reasonable to conclude that this is a bug
in LEDE.

[[https://forum.lede-project.org/t/problems-setting-up-port-forwarding/2037|relevant forum thread]]

[[https://forum.lede-project.org/t/dns-forwarding-issues-past-update-to-lede-reboot-17-01-stable/1962|possibly relevant forum thread]]
